### PR TITLE
Mention create_update_image.sh in Developer Guide

### PR DESCRIPTION
**Pull request by jan-cerny**
The code here was moved to an executable script
create_update_image.sh in pull request #30.

### CODE REVIEW

File: docs/manual/developer_guide.adoc [hunk 1]
```diff
@@ -26,23 +26,10 @@ You probably want to use rhel7-branch. Currently we don't support Fedora (unsupp
 ## Build image
 We will create cpio archive `ASCII cpio archive (SVR4 with no CRC)` packed using gzip (`gzip compressed data`).
 
-You can call this script in oscap-anaconda-addon repository. It will create `update.img` image.
+You can use script `create_update_image.sh` in oscap-anaconda-addon repository. It will create `update.img` image.
```
Comment (redhatrises) [inline docs/manual/developer_guide.adoc]: Thanks @jan-cerny 

Can you change this to:

```You can use the `create_update_image.sh` script in the oscap-anaconda-addon repository to create the `update.img` image.```

Comment (jan-cerny) [reply docs/manual/developer_guide.adoc]: @redhatrises Yes, I can!

**Comment by jan-cerny** [comment]
Changed.

**Comment by redhatrises** [comment]
Thanks. Ack!

**Pull request merged**